examples/playground/client.ts

```diff
@@ -0,0 +1,20 @@
+import { trpc } from '../../src';
+
+import type { RootRouter } from './server';
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export
> import type only imports declarations to be used for type annotations and declarations. It always gets fully erased, so there’s no remnant of it at runtime. Similarly, export type only provides an export that can be used for type contexts, and is also erased from TypeScript’s output.

Reply:
Should be safe for clients